### PR TITLE
fix: Duplicate EJS object keys

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Review by peterbe — COMMENTED** [review]
I don't actually understand which page and what goes wrong in the sidebar?
I also don't understand the edit there to the `LocalizationStatusInSection.ejs` file and how that's related. 

Perhaps @chrisdavidmills can review the actual edit since he's more familiar with the contents of the sidebars.

File: kumascript/macros/LearnSidebar.ejs [hunk 1]
```diff
@@ -81,7 +81,7 @@ var text = mdn.localStringMap({
         'Assessment_typesetting_a_community_school_homepage': 'Assessment: Typesetting a community school homepage',
       'CSS_layout': 'CSS layout',
         'CSS_layout_overview': 'CSS layout overview',
-        'Introduction' : 'Introduction to CSS layout',
+        'CSS_Introduction' : 'Introduction to CSS layout',
```
[the remark this answers is not in the document]
Comment (nschonni) [reply kumascript/macros/LearnSidebar.ejs:84]: Yeah, I had to resist changing it 😄

File: kumascript/macros/LocalizationStatusInSection.ejs [hunk 2]
```diff
@@ -73,7 +73,6 @@ var l10nTranslateNow = mdn.localString({
     "fr"   : "Traduire.",
     "ja"   : "今すぐ翻訳しましょう。",
     "ru"   : "Перевести.",
-    "fr"   : "Traduire.",
```
[the remark this answers is not in the document]
Comment (nschonni) [reply kumascript/macros/LocalizationStatusInSection.ejs:76]: Same string on line 73

**Review by chrisdavidmills — CHANGES_REQUESTED** [review]
Nice work spotting this. I wouldn't mind changing `CSS_Introduction` to `Layout_introduction`, because this particular introduction is about CSS layout, not CSS in general. And we should stick to the convention of sentence casing rather than capitalizing all words.

Which means that `Django_Introduction` should also be changed to `Django_introduction`.

**Comment by mattiapontonio** [comment]
👛
If in the future we use numbers in these cases we must not keep the keys and _discuss etitoriality in an object_.

**Review by chrisdavidmills — APPROVED** [review]
Great stuff, thanks @nschonni !

**Pull request merged**